templates/restaurant_detail.html

```diff
@@ -24,13 +23,13 @@
                                 </div>
                             {% endfor %}
                         </div>
-                        <!-- Slider indicators -->
+                    <!-- Slider indicators -->
```

Comment:
Avoid cosmetic changes like these in templates because they distract the reviewer. You can directly merge these kinds of changes to master. For now it is okay.